This fixes a test failure on SLE11SP3:
Error is:
AssertionError: error: must supply either home or prefix/exec-prefix -- not both

Index: setuptools-15.0/setuptools/tests/test_egg_info.py
===================================================================
--- setuptools-15.0.orig/setuptools/tests/test_egg_info.py
+++ setuptools-15.0/setuptools/tests/test_egg_info.py
@@ -61,7 +61,7 @@ class TestEggInfo:
         )
         cmd = [
             'install',
-            '--home', env.paths['home'],
+            '--prefix', env.paths['home'],
             '--install-lib', env.paths['lib'],
             '--install-scripts', env.paths['scripts'],
             '--install-data', env.paths['data'],
